src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        double area = Math.PI * Math.pow(radius, 2);
+        return Math.round(area);
```

Comment:
The method `calculateArea` is rounding the area value. It's not clear why rounding is performed here; it could lead to a loss of precision. Unless specified, consider returning the exact area value.